exercises/practice/matrix/.meta/zcl_matrix.clas.abap

```diff
@@ -0,0 +1,53 @@
+CLASS zcl_matrix DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+    TYPES integertab TYPE TABLE OF i.
+    METHODS matrix_row
+      IMPORTING
+        string        TYPE string
+        index         TYPE i
+      EXPORTING
+        VALUE(result) TYPE integertab.
```

Comment:
change this to RETURNNIG, 

for this to work, the 
TYPES integertab TYPE TABLE OF i.
must be changed to
TYPES integertab TYPE STANDARD TABLE OF i WITH EMPTY KEY.

Reply:
fixed